portal/src/components/Examples/SimpleForm.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useLayoutEffect, useContext, useState } from "react";
+import { AnimatePresence } from "framer-motion";
+import { TertiaryButton } from "@fremtind/jkl-button-react";
+import { ExampleContext } from "@fremtind/jkl-portal-components";
+
+import PersonalInfoForm from "./SimpleForm/PersonalInfoForm";
+import AddressForm from "./SimpleForm/AddressForm";
+import HouseDetailsForm from "./SimpleForm/HouseDetailsForm";
+
+const SimpleForm = () => {
+    const [showAddress, setShowAddress] = useState(false);
+    const [showHouseDetails, setShowHouseDetails] = useState(false);
+    const { registerKnobs } = useContext(ExampleContext);
+
+    useLayoutEffect(() => {
+        if (registerKnobs) {
+            registerKnobs([
+                <TertiaryButton onClick={() => location.reload()} key="reset">
+                    Reset
+                </TertiaryButton>,
+            ]);
```

Comment:
Synes det er _litt_ skummelt å la folk legge inn hva som helst som knobs hvis dette skal brukes generelt/som komponenteksempler. Men funker fint her!

Reply:
Jeg liker det, da får man mer frihet til å vise hva komponenten din duger til :)

Reply:
Ja, enig at det gir mange bra muligheter! Er bare litt redd for at det kan ende med å bli mye rart i eksemplene hvis det er opp til utvikler å implementere alle knobs manuelt. Så kanskje en kombinasjon? Men det kan vi se på når vi evt. slår sammen eksemplene 😊